libc-test/build.rs

```diff
@@ -2783,6 +2783,9 @@ fn test_freebsd(target: &str) {
             // Added in FreeBSD 14.0
             "TCP_FUNCTION_ALIAS" if Some(14) > freebsd_ver => true,
 
+            // Removed in FreeBSD, deprecated in libc
```

Comment:
You might want to add the release number to this comment

Reply:
Good idea, added